ui/ui.go

```diff
@@ -460,11 +460,18 @@ func (m *Model) setCurrentViewSections(newSections []section.Section) {
 }
 
 func (m *Model) switchSelectedView() config.ViewType {
-	if m.ctx.View == config.PRsView {
-		return config.IssuesView
-	} else {
-		return config.PRsView
+	// get the view excluding the disabled tabs
+	tabs := config.GetViewTabs(m.ctx.Config.Defaults)
+
+	for i, tab := range tabs {
+		// if there are tabs after the selected tab then return it
+		if m.ctx.View == tab.Name && len(tabs)-1 > i {
+			return tabs[i+1].Name
+		}
 	}
```

Comment:
I am not quite getting how to implement with "modulo" that you mentioned above

I was thinking of something like, to loop over the available views(more than 2) and get the next view, if there are none then get the first one in the list(like a circular list), is there something I am missing?

Reply:
@rambhosale I think you can just replace the `for` loop with the code I wrote:

```go
return tabs[(i+1)%len(tabs)];
```

The `config.GetViewTabs` function you wrote, only returns the enabled views.
When you do `(i+1)%len(tabs)`, if:
- `i + 1 < len(tabs)` then you get `i+1`, which is the next tab, like you want
- `i + 1 == len(tabs)` then you get `0` since `x % x == 0`.

Reply:
I am fairly new to Go
So correct me if I am wrong here, in order to get the index of the currently selected view in the Map of enabled views we do have to loop right, since there is no short method for getting the index of a map by its value.

basically for finding the ```i``` in ```tabs[(i+1)%len(tabs)];``` we do have to loop right?
And if we do loop over the list it beats the purpose of eliminating the range?

Reply:
Ah right! I think if I made it so ViewType is a number, then it would have solved this and made the code a bit simpler.
E.g:
```go
type ViewType int
const (
	PRsView    ViewType = iota
	IssuesView ViewType =
)
```
You can go ahead with this code.

Also, wdyt of just hiding the view switcher if there's only 1 view?

Reply:
Got it, no problem!